packages/imperative.ts

```diff
@@ -38,6 +38,76 @@ const config: IImperativeConfig = {
             logFile: Constants.LOG_LOCATION
         }
     },
+    baseProfile: {
+        type: "base",
+        schema: {
+            type: "object",
+            title: "Base Profile",
+            description: "Base profile that stores values shared by multiple service profiles",
+            properties: {
+                host: {
+                    type: "string",
+                    optionDefinition: Constants.BASE_OPTION_HOST
+                },
+                port: {
+                    type: "number",
+                    optionDefinition: Constants.BASE_OPTION_PORT
+                },
+                user: {
+                    type: "string",
+                    secure: true,
+                    optionDefinition: Constants.BASE_OPTION_USER
+                },
+                password: {
+                    type: "string",
+                    secure: true,
+                    optionDefinition: Constants.BASE_OPTION_PASSWORD
+                },
+                rejectUnauthorized: {
+                    type: "boolean",
+                    optionDefinition: Constants.BASE_OPTION_REJECT_UNAUTHORIZED
+                },
+                tokenType: {
+                    type: "string"
+                },
+                tokenValue: {
+                    type: "string",
+                    secure: true
+                },
+                serviceIds: {
```

Comment:
How we could or should use serviceIds still feels like a subject that our group needs to clarify.
Again, nothing to change at this time.